modules/apps/remote-app/remote-app-client-js/src/main/resources/META-INF/resources/js/index.ts

```diff
@@ -320,8 +320,20 @@ function serializeBody(body: BodyInit | null): StructuredClonable {
 		isArrayBufferView(body)
 	) {
 		return body;
+	} else if (body instanceof FormData) {
+
+			const serializedFormData: {[key: string]: File | string} = {};
+
+			body.forEach((value: File | string, key: string) => {
+				serializedFormData[key] = value;
+			});
+
+			return {
+				__FORM_DATA__: serializedFormData,
+			};
+
 	} else {
-		// warn about unserializable body type (eg. FormData, search params,
+		// warn about unserializable body type (eg. search params,
```

Comment:
May as well fix the `else` above while you're at it — newline.

In fact, better go through the whole file and fix all the `else`, `else if` and `catch`. Brian let this sneak through originally, but it's better to not risk him rejecting it.

Reply:
you're right, I'll manually format those